src/commands/data/import/tree.ts

```diff
@@ -6,93 +6,54 @@
  */
 
 import { Messages } from '@salesforce/core';
-import { getString, JsonMap } from '@salesforce/ts-types';
 import { SfCommand, Flags, arrayWithDeprecation } from '@salesforce/sf-plugins-core';
-import type { ImportResult } from '../../../api/data/tree/importTypes.js';
-import { ImportApi, ImportConfig } from '../../../api/data/tree/importApi.js';
+import { importFromPlan } from '../../../api/data/tree/importPlan.js';
+import { importFromFiles } from '../../../api/data/tree/importFiles.js';
 import { orgFlags } from '../../../flags.js';
+import type { ImportResult } from '../../../api/data/tree/importTypes.js';
 
 Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
 const messages = Messages.loadMessages('@salesforce/plugin-data', 'tree.import');
 
 /**
  * Command that provides data import capability via the SObject Tree Save API.
  */
-export default class Import extends SfCommand<ImportResult[] | JsonMap> {
+export default class Import extends SfCommand<ImportResult[]> {
   public static readonly summary = messages.getMessage('summary');
   public static readonly description = messages.getMessage('description');
   public static readonly examples = messages.getMessages('examples');
-  public static readonly aliases = ['force:data:tree:import'];
+  public static readonly aliases = ['force:data:tree:import', 'data:tree:beta:import'];
   public static readonly deprecateAliases = true;
 
   public static readonly flags = {
     ...orgFlags,
     files: arrayWithDeprecation({
       char: 'f',
       summary: messages.getMessage('flags.files.summary'),
-      exclusive: ['plan'],
+      exactlyOne: ['files', 'plan'],
       aliases: ['sobjecttreefiles'],
       deprecateAliases: true,
     }),
     plan: Flags.file({
       char: 'p',
       summary: messages.getMessage('flags.plan.summary'),
+      exactlyOne: ['files', 'plan'],
       exists: true,
     }),
-    'content-type': Flags.string({
-      char: 'c',
-      summary: messages.getMessage('flags.content-type.summary'),
-      hidden: true,
-      aliases: ['contenttype'],
-      deprecateAliases: true,
-      deprecated: { message: messages.getMessage('flags.content-type.deprecation') },
-    }),
-    // displays the schema for a data import plan
-    'config-help': Flags.boolean({
-      summary: messages.getMessage('flags.config-help.summary'),
-      aliases: ['confighelp'],
-      deprecateAliases: true,
-      hidden: true,
-      deprecated: { message: messages.getMessage('flags.config-help.deprecation') },
-    }),
   };
 
-  public async run(): Promise<ImportResult[] | JsonMap> {
```

Comment:
why are we changing so much of these commands for the GA? shouldn't this PR just be changing command files/states/warnings?

Reply:
the command that was `src/commands/data/import/tree.ts` moved to `src/commands/data/import/legacy/tree.ts` and the command that was `src/commands/data/import/beta/tree.ts` moved here.

git doesn't know we swapped the files so it tries to diff this file